src/session/src/context.rs

```diff
@@ -169,6 +179,10 @@ impl QueryContext {
         self.extension.get(key.as_ref()).map(|v| v.as_str())
     }
 
+    pub fn to_extension(&self) -> HashMap<String, String> {
```

Comment:
```suggestion
    pub fn extensions(&self) -> HashMap<String, String> {
```

Reply:
Maybe `extension()`? As the name of the field is `extension`.

It is better to rename the field to `extensions`? @MichaelScofield

Reply:
I think you can't, because Rust dont support function overloading.

Reply:
@evenyag me ok

Reply:
fixed.